OpenRA.Mods.Common/Traits/SpawnActorOnDeath.cs

```diff
@@ -57,6 +61,7 @@ public SpawnActorOnDeath(ActorInitializer init, SpawnActorOnDeathInfo info)
 		{
 			this.info = info;
 
+			crushable = init.Self.TraitOrDefault<Crushable>();
```

Comment:
This may not work when the trait ordering doesn't match `Crushable` before `SpawnActorOnDeath` so either follow @pchote's advice or do the trait lookup in `Killed`.